templates/organization/_help.html.twig

```diff
@@ -0,0 +1,26 @@
+<div>
```

Comment:
this div is useless here

Reply:
Whoups, bad VueJS habit :blush: 
Fixed https://github.com/crf-devs/resop/pull/270/commits/aafc3d260bed68896f43134fad04fbe06927498c